src/Microsoft.AspNet.Mvc.Core/Controllers/ControllerActionDescriptor.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNet.Mvc.Controllers
 {
-    [DebuggerDisplay("CA {DisplayName}(RC-{RouteConstraints.Count})")]
+    [DebuggerDisplay("{DisplayName}")]
```

Comment:
There was really no point in the extra stuff we were putting here, this is much cleaner.